app/main.py

```diff
@@ -16,4 +16,14 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+variables = [lucky_number, pi, one_is_a_prime_number, name, my_favourite_films, profile_info, marks, collection_of_coins]
+
+sorted_variables = {"mutable": [], "immutable": []}
+
+for value in variables:
+
+    if isinstance(value, (list, set, dict)):
```

Comment:
The current type checking does not include tuples as mutable types. Tuples are immutable, so they should be categorized under 'immutable'. The code currently handles this correctly, but ensure that the logic aligns with the intended categorization.